### PR TITLE
[Python] Fixed bug in enum namespace docs

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

**Comment by amarzavery** [comment]
LGTM

**Comment by hovsepm** [comment]
@azuresdkci retest this please

**Comment by annatisch** [comment]
@amarzavery - it seems that there's around 8 tests for serializing/deserializing decimal objects that are generated by the test swagger, but aren't implemented in the test service.
Is there a reason for this or shall I try to add them in?

**Comment by amarzavery** [comment]
@annatisch - The decimal support was initially added by a customer using AutoRest. We did not have proper contribution guidelines at that time. It is possible that the PR got checked in with in adequate test coverage. Please feel free to implement them in the test server and add python tests as well. Please open an issue for test coverage for other languages for tracking purpose.

**Comment by annatisch** [comment]
@amarzavery - awesome, thanks. Will do.

File: AutoRest/TestServer/server/routes/number.js [hunk 1]
```diff
@@ -25,8 +25,21 @@ var number = function(coverage) {
             } else {
                 utils.send400(res, next, "Did not like the value provided for big double in the req " + util.inspect(req.body));
             }
```
Comment (annatisch) [inline AutoRest/TestServer/server/routes/number.js]: Hi @amarzavery,
Let me know if these changes look okay - then once the PR is merged I'll open an issue for other languages :)

File: AutoRest/TestServer/server/routes/number.js [hunk 2]
```diff
@@ -91,8 +124,15 @@ var number = function(coverage) {
             } else {
                 utils.send400(res, next, "Did not like the value provided for small double in the req " + util.inspect(req.params.value));
             }
+		} else if (req.params.format === 'decimal') {
+            if (req.params.value === '2.5976931e-101') {
+                coverage['getDecimalSmall']++
+                    res.status(200).end('2.5976931e-101');
```
Comment (amarzavery) [inline AutoRest/TestServer/server/routes/number.js]: indentation 2 spaces.

File: AutoRest/TestServer/server/routes/number.js [hunk 3]
```diff
@@ -71,8 +97,15 @@ var number = function(coverage) {
             } else {
                 utils.send400(res, next, "Did not like the value provided for small double in the req " + util.inspect(req.body));
             }
+		} else if (req.params.format === 'decimal') {
+            if (req.params.value === '2.5976931e-101' && req.body === 2.5976931e-101) {
+                coverage['putDecimalSmall']++
```
Comment (amarzavery) [inline AutoRest/TestServer/server/routes/number.js]: missing semicolon

File: AutoRest/TestServer/server/routes/number.js [hunk 4]
```diff
@@ -91,8 +124,15 @@ var number = function(coverage) {
             } else {
                 utils.send400(res, next, "Did not like the value provided for small double in the req " + util.inspect(req.params.value));
             }
+		} else if (req.params.format === 'decimal') {
+            if (req.params.value === '2.5976931e-101') {
+                coverage['getDecimalSmall']++
```
Comment (amarzavery) [inline AutoRest/TestServer/server/routes/number.js]: here too ;

**Comment by amarzavery** [comment]
@annatisch - Overall the changes look good. Nit fixes like indentation and semicolon. Once updated, feel free to merge :)

**Comment by annatisch** [comment]
@amarzavery,
Thanks! Changes made - though I don't have the authorization to merge pull requests.
(Also travis seems to be tripping up on something unrelated...)

**Comment by tbombach** [comment]
Merged

**Pull request merged**